### PR TITLE
Fixtures

### PR DESCRIPTION
**Pull request by chouaibj**

### CODE REVIEW

**Review by joelmayemba — APPROVED** [review]

File: src/DataFixtures/ProductFixtures.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        ];
+        foreach ($products as $productData) {
+            $product = new Product();
+            $product->setName($productData['name']);
+            $product->setPrice($productData['price']);
+            $product->setDetail($productData['detail']);
+            $product->setImage($productData['image']);
+            $categoryReference = $this->getReference('category_' . $productData['categoryReference']);
+            $product->setCategory($categoryReference);
+            // $product->setCategory($productData['category']);
```
Comment (Benja-san) [inline src/DataFixtures/ProductFixtures.php]: remove dead code

File: src/DataFixtures/ProductFixtures.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            "Boisson débordante de fraîcheur avec sa Mangue verte...", "image" =>
+            "yanaStart.png", "categoryReference" => "Fruits d'excellences"],
+        ];
+        foreach ($products as $productData) {
+            $product = new Product();
+            $product->setName($productData['name']);
+            $product->setPrice($productData['price']);
+            $product->setDetail($productData['detail']);
+            $product->setImage($productData['image']);
+            $categoryReference = $this->getReference('category_' . $productData['categoryReference']);
```
Comment (Benja-san) [inline src/DataFixtures/ProductFixtures.php:61]: does the reference work ? even if there are spaces and special characters in the string ?

File: templates/base.html.twig [hunk 3]
```diff
@@ -1,34 +1,44 @@
...
+        <script src="assets/js/navbar.js"></script>
+        {{
+            encore_entry_script_tags("app")
+        }}
+        {% endblock %}
+    </head>
+    <body>
+        {% block navbar %} {% include "partials/_navbar.html.twig" %} {%
+        endblock %} {% block content %}
+        <main></main>
```
Comment (Benja-san) [inline templates/base.html.twig]: wait for your PR "update Mocktail controller and index" to be merged on dev then pull dev on this branch to remedy the main tag issue (or solve it here)

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/DataFixtures/ProductFixtures.php (hunk 2 above)
Comment (chouaibj) [reply src/DataFixtures/ProductFixtures.php:61]: yes it works

**Review by Benja-san — APPROVED** [review]

**Review by joelmayemba — APPROVED** [review]

**Pull request merged**